Firestore/Source/Public/FIRFirestore.h

```diff
@@ -137,7 +137,8 @@ NS_SWIFT_NAME(Firestore)
 
 /** Enables or disables logging from the Firestore client. */
 + (void)enableLogging:(BOOL)logging
-    DEPRECATED_MSG_ATTRIBUTE("Use FIRSetLoggerLevel(FIRLoggerLevelDebug) to enable logging");
+    DEPRECATED_MSG_ATTRIBUTE("Use FirebaseConfiguration.shared.setLoggerLevel(_:) to set the "
```

Comment:
In an effort to let people just copy/paste, can you do `FirebaseConfiguration.shared.setLoggerLevel(.debug)` ?

Reply:
Okay, should I change the text to, "to enable logging" as well instead of "to set the appropriate logging level"?

Reply:
Yup, that would be my preference. :-)